uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -977,30 +971,25 @@ define([
       }
 
       /**
-       * Loads just one pre-selected managee
-       *
-       * @return {Promise}
-       */
-      function loadManagee () {
-        return Contact.find(this.directiveOptions.selectedContactId)
-          .then(function (contact) {
-            this.managedContacts = [contact];
-          }.bind(this));
-      }
-
-      /**
        * Loads the managees of currently logged in user
        *
        * @return {Promise}
        */
       function loadManagees () {
-        return Contact.find(this.directiveOptions.contactId)
-          .then(function (contact) {
-            return contact.leaveManagees();
-          })
-          .then(function (contacts) {
-            this.managedContacts = contacts;
-          }.bind(this));
+        if (this.directiveOptions.selectedContactId) {
```

Comment:
I was about to explain the reasoning behind my solution but then I realized I might have had the wrong assumption. The logic in my solution was that we had to make sure that the given "selected contact" was indeed among the managees of the currently logged in contact, but given that the currently logged in contact in this case is an admin, then the admin is allowed to "use" that contact regardless if he's the contact's l&a manager or not

I think then your solution works in this context